package.json

```diff
@@ -13,16 +15,43 @@
     "url": "git+https://github.com/matthew1232/stockx-api.git"
   },
   "keywords": [
-    "stockx",
-    "api",
-    "promise",
+    "StockX",
+    "sneakers",
     "bots",
-    "request"
+    "axios"
```

Comment:
```suggestion
    "requuest",
    "promise",
    "es6"
```

Reply:
Pretty sure that's a typo, "requuest" should be "request"

Reply:
Whoops! Good catch.